Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM clearlinux/mixer-ci:latest
 ENV LC_ALL="en_US.UTF-8"
```

Comment:
Can't you set it here as well? Or do you need it set for that one command only?

Reply:
still trying to just get travis/docker to pass. I'll comment when this is actually ready for review again.